cfgov/v1/management/commands/delete_page_cache.py

```diff
@@ -4,15 +4,16 @@
 
 
 class Command(BaseCommand):
-    help = "Delete the cache of a page by its full URL"
+    help = "Delete the cache of pages by full URLs"
 
     def add_arguments(self, parser):
         parser.add_argument(
             "--url",
             required=True,
-            help="The full URL for the page you wish to delete its cache",
+            help="The full URLs for pages you wish to delete its cache",
```

Comment:
Could we use `nargs="+"` to avoid the need to enclose the URLs in quotes, so you can do this:

```sh
$ cfgov/manage.py delete_page_cache --url foo bar ...
```

instead of having to do this:

```sh
$ cfgov/manage.py delete_page_cache --url "foo bar ..."
```

```suggestion
            nargs="+",
            help=(
                "The full URL for the page cache to delete "
                "(can specify multiple)"
            )
```

This then shows up in the `--help` output like this:

```sh
optional arguments:
  -h, --help            show this help message and exit
  --url URL [URL ...]   The full URLs for the page cache to delete (can
                        specify multiple)
  ...
```

Reply:
Thats is a great suggestion to avoid enclosing the URLs in quotes.